### PR TITLE
Several policy updates: consistency with further discussion on other …

### PR DESCRIPTION
**Pull request by Pinolo**
…sections, more clarity

### CODE REVIEW

File: reference/policy.md [hunk 1]
```diff
@@ -4,51 +4,51 @@
...
-* [Policy](./policy.md)  
+* [People](./people.md)
+* [Process](./process.md)
+* [Policy](./policy.md)
 * [Technology](./technology.md)
 
 ## Introduction
 
-The Cloud Native Maturity Model covers four major dimensions - People, Process, Policy and Technology. This paper addresses policy. Your goal of cloud native should be to implement policy to embrace shift left security. Across all levels, you will seek to implement policies and automate enforcement in CI/CD and continuously monitor against policies in production.
+The Cloud Native Maturity Model covers four major dimensions - People, Process, Policy and Technology. This paper addresses policy, the critical implementation of internal requirements, and compliance to external regulations. Your goal as a cloud native practitioner should be to implement policies that enable and encourage shift left security. As your maturity grows, you will seek to automate policy enforcement in CI/CD and continuously monitor against policies in production.
```
Comment (Pinolo) [inline reference/policy.md:14]: "shift left" with or without capitals? I'm confused by even the CNCF Glossary using both forms with no apparent reasons.

**Comment by Pinolo** [comment]
@daniellemcook @siforster @Deep-Cyber @rglenn-accenture: although we didn't propose any changes to "Policy" during the review, I made some, based on the rationale developed during the following reviews (e.g. avoid prescriptive product/practice mentions). I also tried to clarify and expand a little on what I found needing more context or reasoning.

Please don't be afraid to cut my edits short, and do suggest any language improvements that you deem appropriate. Thanks!

**Comment by rglenn-accenture** [comment]
I think I have to be on my personal account to formally review, but I like these updates a lot! I would approve these changes.

**Review by rglenn-accenture — APPROVED** [review]
I like all of these changes. They provide great clarity.

**Pull request merged**